src/main/java/net/sf/jabref/gui/BasePanel.java

```diff
@@ -1840,17 +1842,6 @@ private synchronized void markChangedOrUnChanged() {
         frame.setWindowTitle();
     }
 
-    /**
-     * Selects a single entry, and scrolls the table to center it.
-     *
-     * @param pos Current position of entry to select.
-     */
-    public void selectSingleEntry(int pos) {
```

Comment:
nope, this was an unused method which did almost the same as `public void highlightEntry(final BibEntry be)` (accepting a position instead of a `BibEntry`